manifests/app-proxy/kustomization.yaml

```diff
@@ -3,7 +3,7 @@ kind: Kustomization
 images:
   - name: quay.io/codefresh/cap-app-proxy
     newName: quay.io/codefresh/cap-app-proxy
-    newTag: v0.0.28
+    newTag: v0.0.29
```

Comment:
I see that this tag doesn't exist in quay yet, please make sure to push it before bumping